### DISCUSSION TITLE
Show display type in HTML

### DISCUSSION THREAD

**Discussion by benoitblancquaert** [verbb/formie#1196] [Ideas]
Hi!

I was wondering how to style a form field (of type _entries_) based on the field display settings. There is no class or data attribute for this. Would it be possible to show the display type (_dropdown_, _checkboxes_ or _radio buttons_) via a data attribute? This would be super handy.

For example, a form field of type _entries_:

```
<div class="fui-field fui-type-entries fui-label-above-input fui-subfield-label-above-input fui-instructions-above-input fui-page-field" data-field-handle="test-field" data-field-type="entries">
    ...
</div>
```

It would be helpful if the HTML above had an additional data attribute that matches the field display settings like `data-display-type="dropdown"`.

Thanks for looking into this.

**Comment by engram-design** [comment]
A fair enough suggestion! Added for the next release (`data-field-display-type`). To get fix early, change your `verbb/formie` requirement in `composer.json` to:

```
"require": {
  "verbb/formie": "dev-craft-4 as 2.0.18",
  "...": "..."
}
```

Then run `composer update`.
